android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="app.revanced.manager.flutter">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
```

Comment:
Are you using foreground services?

Android 14 requires you to specify types and additional permissions: https://developer.android.com/about/versions/14/changes/fgs-types-required

Reply:
It seems like it was not using the Foreground Service permission. So I removed it from the manifest.